eslint.config.js

```diff
@@ -0,0 +1,17 @@
+import pluginVue from "eslint-plugin-vue";
+import eox from "@eox/eslint-config";
+
+export default [
+  ...eox,
+  ...pluginVue.configs["flat/essential"],
+  {
+    ignores: ["public/", "dist/", ".eodash/"],
+  },
+  {
+    files: ["core/client/**/*.vue", "widgets/**/*.vue"],
+    rules: {
+      "vue/no-deprecated-html-element-is": "warn",
```

Comment:
we use some deprecated elements?

Reply:
we use `animated-details` web component to animate the `WidgetContainer` and it's assigned using the `is` property (that is apparently deprecated now)  in a `detail` element.

Reply:
interesting, ok, we can keep it as warning as a reminder this is something we should look into in the future, thanks